cli/openApiSpec.ts

```diff
@@ -0,0 +1,270 @@
+import fs from 'fs'
+import { Command, Option } from "commander";
+import { ValidatorJSON } from 'convex/values'
+import chalk from 'chalk';
+import { AnalyzedFunction, FunctionSpec, getFunctionSpec } from './utils';
+import prettier from "prettier";
+
+export const openApiSpec = new Command("open-api-spec")
+    .summary("Generate an OpenAPI spec from a Convex function definition")
+    .argument("[filePath]", "The file name of the Convex function definition. If this argument is not provided, we will \nretrieve the function spec from your configured convex deployment.\n"
+        + "The file name defaults to `convex-spec-{msSinceEpoch}`."
+    )
+    .addOption(
+        new Option(
+            "--outputFile <filename>",
+            "Specify the output file name for your spec.",
+        )
+            .default(undefined)
+    )
+    .addOption(
+        new Option(
+            "--public-only",
```

Comment:
Thoughts on inverting this as `--include-internal` and still defaulting to false?

Reply:
that seems reasonable